### PR TITLE
Handled check health error and added new actions and parameters

### PR DESCRIPTION
**Pull request by cs-utkarsh-surana**
#### Descriptions:

- Added the following new parameters in the action `Send Message`
  - Custom String
  - Country
  - List ID
  - Email

- Added the following new operations and respective playbooks
  - Get Contact List
  - Send Voice Message

#### Fix:

- Fixed the connector issue where check health was failing because of the following error - `'cannot import name '_api_call'`

### CODE REVIEW

File: clicksend/info.json [hunk 1]
```diff
@@ -98,20 +136,252 @@
           "visible": true,
           "editable": true,
           "required": false,
-          "tooltip": "When the message should be sent",
-          "description": "Specify the date and time when the message should be sent."
+          "tooltip": "When the message should be sent. Leave blank for immediate delivery.",
+          "description": "Specify the date and time when the message should be sent. Leave blank for immediate delivery."
```
Comment (cs-sameer-pathan) [inline clicksend/info.json:140]: Please make description and tooltip same.

File: clicksend/info.json [hunk 2]
```diff
@@ -98,20 +136,252 @@
...
+        },
+        {
+          "title": "Voice",
+          "name": "voice",
+          "type": "select",
+          "visible": true,
+          "editable": true,
+          "required": true,
+          "tooltip": "Specify 'female' or 'male'.",
+          "description": "Specify 'female' or 'male'.",
```
Comment (cs-sameer-pathan) [inline clicksend/info.json]: Please specify whether you want to use a 'Female' or 'Male' voice to send the voice message.

File: clicksend/info.json [hunk 3]
```diff
@@ -98,20 +136,252 @@
...
+          "options": [
+            "male",
+            "female"
```
Comment (cs-sameer-pathan) [inline clicksend/info.json]: please provide options Male , Female

File: clicksend/info.json [hunk 4]
```diff
@@ -98,20 +136,252 @@
...
+        },
+        {
+          "title": "to",
+          "name": "to",
+          "type": "text",
+          "visible": true,
+          "editable": true,
+          "required": false,
+          "tooltip": "Specify the phone number in E.164 format.",
+          "description": "Specify the phone number in E.164 format."
```
Comment (cs-sameer-pathan) [inline clicksend/info.json]: Specify the phone number of recipient in E.164 format.

File: clicksend/info.json [hunk 5]
```diff
@@ -98,20 +136,252 @@
...
+          "required": true,
+          "tooltip": "Specify 'female' or 'male'.",
+          "description": "Specify 'female' or 'male'.",
+          "options": [
+            "male",
+            "female"
+          ]
+        },
+        {
+          "title": "to",
```
Comment (cs-sameer-pathan) [inline clicksend/info.json]: Change the title to To

File: clicksend/info.json [hunk 6]
```diff
@@ -98,20 +136,252 @@
...
+        {
+          "title": "Machine Detection",
+          "name": "machine_detection",
+          "type": "checkbox",
+          "visible": true,
+          "editable": true,
+          "required": false,
+          "value": false,
+          "tooltip": "Specify whether to detect answering machine or voicemail and leave a message.",
+          "description": "Specify whether to detect answering machine or voicemail and leave a message."
```
Comment (cs-sameer-pathan) [inline clicksend/info.json]: By default this is set to false

File: clicksend/info.json [hunk 7]
```diff
@@ -98,20 +136,252 @@
...
+        },
+        {
+          "title": "Custom String",
+          "name": "custom_string",
+          "type": "text",
+          "visible": true,
+          "editable": true,
+          "required": false,
+          "tooltip": "Specify your reference. It will be passed back with all replies and delivery reports.",
+          "description": "Specify your reference. It will be passed back with all replies and delivery reports."
```
Comment (cs-sameer-pathan) [inline clicksend/info.json:330]: Can you make it descriptive?

File: clicksend/info.json (hunk 1 above)
Comment (cs-utkarsh-surana) [reply clicksend/info.json:140]: done

File: clicksend/info.json (hunk 2 above)
Comment (cs-utkarsh-surana) [reply clicksend/info.json]: Donedone

File: clicksend/info.json (hunk 3 above)
Comment (cs-utkarsh-surana) [reply clicksend/info.json]: done

File: clicksend/info.json (hunk 4 above)
Comment (cs-utkarsh-surana) [reply clicksend/info.json]: done

File: clicksend/info.json (hunk 5 above)
Comment (cs-utkarsh-surana) [reply clicksend/info.json]: done

File: clicksend/info.json (hunk 6 above)
Comment (cs-utkarsh-surana) [reply clicksend/info.json]: done

File: clicksend/info.json (hunk 7 above)
Comment (cs-utkarsh-surana) [reply clicksend/info.json:330]: Will provide additional info to indo dev team to reframe the description properly.

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**